### ISSUE TITLE
Error when setting padding to "strict"

### ISSUE THREAD

**Issue by marcprux** [uwdata/vega#28]
Setting "padding": "strict" in a spec in the Vega 2 editor results in a Javascript error:

```
[Error] TypeError: undefined is not a function (evaluating 'this._model.width(this._width)')
    autopad (vega2.js, line 9979)
    update (vega2.js, line 10112)
    (anonymous function) (editor.js, line 64)
    (anonymous function) (vega2.js, line 10492)
```

**Comment by marcprux** [comment]
It appears that the `width()` and `height()` functions were removed from `Model`. Restoring them to look like Vega 1.5 and then removing the `this._reset.axes = true` line sort of works, but the axes are messed up. Is there some new way to request that the axes get reset?

**Comment by arvind** [comment]
Thanks! Fixed in e8f6e25.
